pyccel/parser/syntactic.py

```diff
@@ -734,8 +735,13 @@ def fill_types(ls):
                     errors.report(msg,
                                 bounding_box = (stmt.lineno, stmt.col_offset),
                                 severity='warning')
-
                 templates[tp_name] = hdr_parse(stmts=txt)
+                template['templates_dict'] = templates
+                template['template_list'] = decorators['template']
```

Comment:
Could you change templates_dict and template_list to something a bit more self explanatory

Reply:
any suggestions ?